launch/state_machines/challenge_spr.launch

```diff
@@ -23,9 +23,11 @@
         </group>
 
         <!--Launch SkyBiometry rosrun skybiometry_ros face_properties_node-->
+<!--
         <node pkg="skybiometry_ros" type="face_properties_node" name="skybiometry_face_properties">
                 <param name="api_timeout" value="60" />
         </node>
+-->
```

Comment:
If you are removing code, please remove it

Reply:
Oh that was the plan